torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6357,11 +6357,14 @@ def sample_inputs_spectral_ops(self, device, dtype, requires_grad=False, **kwarg
         # cuFFT supports powers of 2 for half and complex half precision
         # NOTE: For hfft, hfft2, hfftn, irfft, irfft2, irfftn with default args
         # where output_size n=2*(input_size - 1), we make sure that logical fft size is a power of two
-        if self.name in ['fft.hfft', 'fft.irfft']:
+        if self.name in ['fft.hfft', 'fft.irfft',
```

Comment:
nit: This is why I prefer partialing the sample inputs functions with kwargs that control its behavior vs. matching the actual name exactly, but nbd